service_config/test_java_components.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# clone Qanary pipeline
+git clone https://github.com/WDAqua/Qanary.git
```

Comment:
@heinpa Seems to be not required anymore due to the Qanary packages stored at Maven repository, now.